manim/mobject/mobject.py

```diff
@@ -96,6 +123,39 @@ def __init__(self, color=WHITE, name=None, dim=3, target=None, z_index=0, **kwar
 
         Container.__init__(self, **kwargs)
 
+    @classmethod
+    def animation_override_for(cls, animation_class):
+        if animation_class in cls.animation_overrides:
+            return cls.animation_overrides[animation_class]
+
+        return None
```

Comment:
Not returning anything means returning `None`, so I think this is redundant.

Reply:
I think it's better to explicitly return `None` as to me, not returning anything means "it doesn't make sense to use the return value", but explicitly returning `None` signals that `None` is a valid, intentional return value that participates in the logic.